tests/e2e/e2e_test.go

```diff
@@ -98,6 +98,26 @@ var _ = Describe("odo", func() {
 		})
 	})
 
+	Context("creating component without an application", func() {
+		It("should create the component in default application", func() {
+			runCmd("odo create php testcmp")
+
+			getCmp := runCmd("odo component get --short")
+			Expect(getCmp).To(Equal("testcmp"))
+
+			getApp := runCmd("odo app get --short")
+			Expect(getApp).To(Equal("app"))
+		})
+
+		// TODO: Failing. Uncomment when fixed.
```

Comment:
why is this failing?

Reply:
@kadel  It is working now. Have uncommented it and pushed the changes.